### PR TITLE
Premium content: Improve editing consistency

### LINKED ISSUE #22425: Premium Content Block: Same Elements Labels and Order Across the Editor
### Impacted plugin

Jetpack

### What

Make the Premium Content block UI more consistent.

### How

The Premium Content block comprises two views, and they are not presented consistently across the editor UI.

| Block Toolbar | List View |
| - | - |
| Screenshot 2022-01-20 at 11 44 29 | Screenshot 2022-01-20 at 11 44 36 |

The view displaying content for logged out visitors is labelled as "visitor" in the block toolbar and "logged out" in the list view.
We should pick one.

They are also presented in different orders: [Visitor - Subscriber] in the block toolbar, and [Subscriber - Logged Out] in the list view.
Again, we should pick one.

---

See: p58i-bIN-p2 (relevant part starts at around minute 9)

### PR DESCRIPTION
**Pull request by dsas**
In the editor, viewing the content for people who don't have a
subscription is sometimes referred to as 'Visitor view' and sometimes as
'Logged out view'. This change makes the interface more consistent by using 'Guest View'.

Swap order of visitor/subscriber buttons in the editor to match the
order of the added content.

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

1. Apply patch - on simple by using the generated wpcom diff, or using a jurassic ninja and jp beta with feature branch
2. Create a new post
3. Insert a premium content block

Thing | Before | After
----|-------|------
Block toolbar | image |  image
List view | image | image

Fixes #22425

### CODE REVIEW

File: projects/plugins/jetpack/extensions/blocks/premium-content/edit.js [hunk 1]
```diff
@@ -332,21 +332,21 @@ function Edit( props ) {
 				<ToolbarGroup>
```
Comment (dsas) [inline projects/plugins/jetpack/extensions/blocks/premium-content/edit.js:332]: This diff looks messier than it is, essentially I cut the Subscriber view and pasted it above the Guest View

**Comment by BogdanUngureanu** [comment]
Great work Dean, it tests well on both Atomic and Simple. There's one thing that looks a bit unexpected though: now, the default selected tab is the one from right (Guest View), which is not common pattern. Maybe we switch order of the blocks?
Screenshot 2022-02-03 at 15 38 38

**Comment by dsas** [comment]
Oh, good eye for detail 👏 @BogdanUngureanu 

I've made that change so that it's consistent.

To me the subscriber content is the primary thing someone might want to work on next, so I thought about making the subscriber view the default and first one. The visitor content already has a good placeholder and probably isn't what you're excited about writing about. I've stuck with the existing flow though - visitor view first, as maybe someone has thought more than me about this in the past, and to avoid disrupting existing users.

**Comment by BogdanUngureanu** [comment]
Works as expected! 🥳 

> To me the subscriber content is the primary thing someone might want to work on next, so I thought about making the subscriber view the default and first one. The visitor guest content already has a good placeholder and probably isn't what you're excited about writing about. I've stuck with the existing flow though - visitor view first, as maybe someone has thought more than me about this in the past, and to avoid disrupting existing users.

Having the content as makes more sense to me too, but I think we should keep the current behaviour for now.

**Review by BogdanUngureanu — APPROVED** [review]

**Comment by Copons** [comment]
I don't have anything against it, but is there a reason why we settled on "Guest" instead of picking one of the existing options? 😄 

I'd say "guest" is a fairly reasonable label, although I wonder if by keeping, say, "Visitor" we'd also keep its translation instead of possibly needing a new one.

(Randomly pinging @apeatling for opinions, because, as per tradition, all blames end with him. :trollface: )

**Comment by dsas** [comment]
> I don't have anything against it, but is there a reason why we settled on "Guest" instead of picking one of the existing options? 

It's traditional when there are two competing standards to unify them by creating another :trollface: 

"Logged Out view" seemed a bit unwieldy to me, I didn't like "Visitor view" because everywhere else on WPcom (stats, dashboard, repeat visitor block) 'Visitor' is just any website viewer, whether subscribed or not. Guest just seemed to get over the idea better to me.

Though now I think about it, 'Guest' isn't strictly accurate as you could be contributing on one plan, but this premium content is gated behind a different plan 🤷‍♂️. Happy to go back to Visitor if that's preferred.

**Comment by dsas** [comment]
Maybe changing to "Content" and something like "Payment prompt" or just "Paywall" makes more sense.

**Comment by Copons** [comment]
@dsas Of all the suggestions, Guest View sounds best to me 😄 
Though, I've added a `Needs Copy Review` in the off-chance editorial folks have another idea.

**Review by jeherve — APPROVED** [review]

**Comment by dsas** [comment]
Changeset r239595-wpcom

**Pull request merged**